docs/src/pages/getting-started/installation/dependencies.flutter.mdx

```diff
@@ -6,10 +6,12 @@ Add the `amplify_authenticator` package with [pub](https://pub.dev/):
 flutter pub add amplify_authenticator
 ```
 
-This will add a dependency like this to your package's pubspec.yaml
+This will add a dependency like this to your package's `pubspec.yaml`
 
 ```yaml
 dependencies:
+  amplify_flutter: ^0.4.0
```

Comment:
Looks good.

Optionally, we could highlight the line for `amplify_authenticator` since that is the only package that will be added by running the command above.

Changing `yaml` to `yaml{4}` will highlight that line.

![Screen Shot 2022-03-01 at 4 35 39 PM](https://user-images.githubusercontent.com/20613561/156253178-9ef50e41-f757-4d40-9e1e-4052230663f8.png)

Reply:
I feel like that would make it seem that this is the only thing you need to make authenticator work, which is not entirely true. What do you think?

Reply:
Yeah, I could see that.

On line 6 it says to run `flutter pub add amplify_authenticator`, but the snippet here shows three dependencies added. The command and the snippet sort of contradict each other. I was thinking the highlight makes it clearer that running `flutter pub add amplify_authenticator` doesn't add all three dependencies.

Maybe we update the lines above to mention the other two packages 

> Add `amplify_flutter`, `amplify_auth_cognito`, and `amplify_authenticator` package with [pub](https://pub.dev/):
> 
> flutter pub add amplify_flutter
flutter pub add amplify_auth_cognito
flutter pub add amplify_authenticator

Reply:
I am thinking maybe we remove that completely - I feel like the convention for a lot of beginner developers is to immediately add the dependencies in the pubspec.yaml file rather than use `flutter pub`. That is also what we recommend all across the library getting started guide, which I think is the more conventional path. What do you think? @Jordan-Nelson